types/ysdk/index.d.ts

```diff
@@ -0,0 +1,358 @@
+        id: string;
+        developerPayload?: string;
+    }): Promise<TSigned extends true ? Signed<Purchase> : Purchase>;
+    consumePurchase(token: string): Promise<void>;
+}
+
+export interface Leaderboards {
+    getLeaderboardDescription(leaderboardName: string): Promise<LeaderboardDescription>;
+
+    setLeaderboardScore(leaderboardName: string, score: number, extraData?: string): Promise<void>;
+
+    /**
+     * @throws {{code: string}}
+     */
+    getLeaderboardPlayerEntry(leaderboardName: string): Promise<LeaderboardEntry>;
+
+    getLeaderboardEntries(
+        leaderboardName: string,
+        opts?: {
+            includeUser?: boolean;
+            quantityAround?: number;
+            quantityTop?: number;
+        },
+    ): Promise<LeaderboardEntriesData>;
+}
+
+export interface LeaderboardEntriesData {
+    leaderboard: LeaderboardDescription;
+    ranges: Array<{ start: number; size: number }>;
+    userRank: number;
+    entries: LeaderboardEntry[];
+}
+
+export interface LeaderboardEntry {
+    score: number;
+    extraData?: string;
+    rank: number;
+    player: {
+        getAvatarSrc(size: "small" | "medium" | "large"): string;
+        getAvatarSrcSet(size: "small" | "medium" | "large"): string;
+        lang: string;
+        publicName: string;
+        scopePermissions: {
+            avatar: string;
+            public_name: string;
+        };
+        uniqueID: string;
+    };
+    formattedScore: string;
+}
+
+export interface LeaderboardDescription {
+    appID: string;
+    default: boolean;
+    description: {
+        invert_sort_order: boolean;
+        score_format: {
+            options: {
+                decimal_offset: number;
+            };
+        };
+        type: "numberic" | "time";
+    };
+    name: string;
+    title: {
+        [lang: string]: string;
+    };
+}
+
+export type FeedbackError = "NO_AUTH" | "GAME_RATED" | "REVIEW_ALREADY_REQUESTED" | "UNKNOWN";
+
+export type StickyAdvError = "ADV_IS_NOT_CONNECTED" | "UNKNOWN";
+
+export type SdkEventName = "EXIT" | "HISTORY_BACK";
+
+/* prettier-ignore */
+export type ISO_639_1 =
+    | "af"
+    | "am"
+    | "ar"
+    | "az"
+    | "be"
+    | "bg"
+    | "bn"
+    | "ca"
+    | "cs"
+    | "da"
+    | "de"
+    | "el"
+    | "en"
+    | "es"
+    | "et"
+    | "eu"
+    | "fa"
+    | "fi"
+    | "fr"
+    | "gl"
+    | "he"
+    | "hi"
+    | "hr"
+    | "hu"
+    | "hy"
+    | "id"
+    | "is"
+    | "it"
+    | "ja"
+    | "ka"
+    | "kk"
+    | "km"
+    | "kn"
+    | "ko"
+    | "ky"
+    | "lo"
+    | "lt"
+    | "lv"
+    | "mk"
+    | "ml"
+    | "mn"
+    | "mr"
+    | "ms"
+    | "my"
+    | "ne"
+    | "nl"
+    | "no"
+    | "pl"
+    | "pt"
+    | "ro"
+    | "ru"
+    | "si"
+    | "sk"
+    | "sl"
+    | "sr"
+    | "sv"
+    | "sw"
+    | "ta"
+    | "te"
+    | "tg"
+    | "th"
+    | "tk"
+    | "tl"
+    | "tr"
+    | "uk"
+    | "ur"
+    | "uz"
+    | "vi"
+    | "zh"
+    | "zu";
+
+/* prettier-ignore */
```

Comment:
please remove that one and other one, DT is now using `dtprint` for formatting (refers to readme for details)